### ISSUE TITLE
drizzle-orm isSQLWrapper

### ISSUE THREAD

**Issue by MigFerreira** [DallasHoff/sqlocal#36]
In file:
`http://127.0.0.1:5173/node_modules/.pnpm/sqlocal@0.11.0/node_modules/sqlocal/dist/lib/normalize-statement.js?v=694ee066`

On first line
`import { isSQLWrapper } from "/@id/__vite-optional-peer-dep:drizzle-orm:sqlocal";`

Getting following error:
```
Uncaught SyntaxError: The requested module 'http://127.0.0.1:5173/@id/__vite-optional-peer-dep:drizzle-orm:sqlocal' doesn't provide an export named: 'isSQLWrapper' normalize-statement.js:1:10
```

I am not using drizzle...
using vite 5.3.5

**Comment by MigFerreira** [comment]
Works again once rolling back to 0.10.2

**Comment by DallasHoff** [comment]
Thanks for the report! Should be fixed in [0.11.1](https://github.com/DallasHoff/sqlocal/releases/tag/v0.11.1)
